### DISCUSSION TITLE
Got a "2FA checkup" on the website

### DISCUSSION THREAD

**Discussion by DevCatalina** [community/community#55289] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey all, recently i got a prompt on the website asking me to do q 2FA checkup by putting my code in, is that normal or did someone access my account? Virtual support did not help me.

**Comment by DNin01** [comment, marked as answer]
Is this the prompt that appeared?
![image](https://i0.wp.com/user-images.githubusercontent.com/1666363/211381828-ce246f95-a714-4bc2-8a9b-6f991b648334.png?ssl=1)

If so, your account is perfectly fine. 2FA checkup happens 28 days after you configure 2FA to ensure that your two-factor method is working correctly. It's meant to help protect against account lockouts. For example, if you set up 2FA using a TOTP app and then, without you realizing, that TOTP app stopped working, you wouldn't be able to authenticate. Fortunately, the 2FA checkup confirms if your two-factor method is working properly, and if it isn't you'll still have access to your account and can reconfigure 2FA to avoid being locked out. You can see this [blog post](https://github.blog/changelog/2023-01-11-second-factor-validation-after-2fa-setup/) for more information.
